### DISCUSSION TITLE
Search Bar - Remove Route & POI

### DISCUSSION THREAD

**Discussion by adrianfilipeu** [FacilMap/facilmap#251] [Q&A] [answered]
Is there any possibility to keep the Search but remove the Route and POI tabs from the bar?

![image](https://github.com/FacilMap/facilmap/assets/35315380/45452bf0-6316-4377-a683-7cdfca559dfd)

Currently playing with the iframe version.

**Comment by cdauth** [comment]
Not right now, but this would be a useful addition.

**Reply by adrianfilipeu** [reply]
Hi @cdauth 
First, many thanks for your development. 
And yes, I also think it would be a very useful addition. 
Some use-cases only require pinning locations on the map, not using routes of POIs.

**Reply by adrianfilipeu** [reply]
Just putting this out there, maybe the crosshair button (current location) could also be manageable on UI. Show/hide based on admin setup.

**Comment by cdauth** [comment, marked as answer]
There is now a `route`, `pois` and `locate` query parameter to control these.

**Reply by adrianfilipeu** [reply]
Amazing! U da best!
